.travis.yml

```diff
@@ -3,9 +3,9 @@ language: python
 env:
   global:
     - CONDA_CHANNELS='astropy-ci-extras astropy http://ssb.stsci.edu/conda-dev'
-    - CONDA_DEPENDENCIES="astropy astroquery matplotlib jupyter jwst"
+    - CONDA_DEPENDENCIES="astropy>=3.1 astroquery matplotlib jupyter jwst webbpsf"
     - PIP_DEPENDENCIES="reproject spectral-cube image_registration tweakwcs photutils"
-    - PYTHON_VERSION=3.6
+    - PYTHON_VERSION=3.7
```

Comment:
Is this absolutely necessary?  This is a bit of an ask for a user given it's still pretty new (but not the end of the world since I think all the conda stuff applies now)

Reply:
No, it was just a test.